web/src/lib/components/admin-page/settings/setting-input-field.svelte

```diff
@@ -39,8 +40,14 @@
 			</div>
 		{/if}
 	</div>
+
+	<div class={`flex place-items-center gap-1`}>
+		<label class={`immich-form-label text-xs`} for="{label}-desc">{desc}</label>
+	</div>
+
 	<input
-		class="immich-form-input w-full"
+		style="position:relative;top:10px"
+		class="immich-form-input gap-2 w-full"
```

Comment:
Also for `SettingSelect` component

```suggestion
		class="immich-form-input w-full"
		aria-describedby={desc ? `${label}-desc` : undefined}
```